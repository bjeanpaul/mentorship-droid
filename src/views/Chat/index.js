import React from 'react';
import { BaseView } from 'src/components';

const Chat = () => (
  <BaseView />
);


Chat.propTypes = {
};


export default Chat;
