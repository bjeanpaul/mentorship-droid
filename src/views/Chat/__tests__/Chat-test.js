import React from 'react';

import Chat from 'src/views/Chat';


describe('Chat', () => {
  function createComponent(props = {}) {
    return (
      <Chat
        {...props}
      />
    );
  }

  it('should render', () => {
    expect(render(createComponent())).toMatchSnapshot();
  });
});
