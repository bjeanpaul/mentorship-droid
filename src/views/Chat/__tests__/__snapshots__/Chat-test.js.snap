exports[`Chat should render 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#fff",
        "flex": 1,
      },
      undefined,
    ]
  } />
`;
